src/paneer-tikka-masala.md

```diff
@@ -0,0 +1,46 @@
+# Paneer Tikka Masala
+
+![paneer tikka masala](pix/paneer-tikka-masala-00.webp "Paneer Tikka Masala served with Naan bread")
+
+## Ingredients
+
+- 250g cubes paneer
+- 1 onion
+- 1 capsicum
+- 200ml tomato puree 
+- 1 tsp red chilli powder
+- 1 tsp turmeric powder
+- 1 tsp coriander powder
+- 1/2 tsp garam masala powder
+- 100ml thick yogurt
+- 1 tsp fenugreek powder
+- dry fenugreek leaves
+- cardamom
+- cumin seeds
+- shaved ginger
+- 20ml garlic paste
+- 1 bowl corn
+
+## Directions
+
+1. Take a glass container, and mix the cubed paneers with half a tea spoon of red chilli powder, turmeric powder, coriander powder, garam masala powder, fenugreek powder, and 50g yogurt.
+2. Let the mix marinade for at least an hour.
+ 
+![paneer tikka masala](pix/paneer-tikka-masala-01.webp "Paneer ready to be marinated on the skewer")
+
+3. Preheat the oven to 200°C (392°F), and place the marinated paneer cubes, with onion, capsicum, and tomato on the skewers. Let them grill in the oven using skewer support, by continuously flipping them every 5 minutes until the outer crust looks crispy enough.
```

Comment:
```suggestion
2. Let the mix marinade for at least an hour.
![paneer tikka masala](pix/paneer-tikka-masala-01.webp "Paneer ready to be marinated on the skewer")
3. Preheat the oven to 200°C (392°F), and place the marinated paneer cubes, with onion, capsicum, and tomato on the skewers. Let them grill in the oven using skewer support, by continuously flipping them every 5 minutes until the outer crust looks crispy enough.
```

Annoying thing about markdown is that if you leave an empty space between items in a numbered list, it actually restarts the count back to 1, so this would go:

```
1. .....
2. .....
image here
1. .....
2.  .....
```

So to avoid this, just remove the empty lines before and after the image

Reply:
Sorry, should have mentioned this in my last comment about numbering.